algo-coding-exercise/backtest/src/test/java/codingblackfemales/backtest/AddCancelAlgoBackTest.java

```diff
@@ -117,16 +120,17 @@ public void testExampleBackTest() throws Exception {
         //create a sample market data tick....
         send(createSampleMarketDataTick());
         //simple assert to check we had 3 orders created
-        //assertEquals(container.getState().getChildOrders().size(), 3);
+        assertEquals(container.getState().getChildOrders().size(), 21);
 
         //when: market data moves towards us
         send(createSampleMarketDataTick2());
 
         //then: get the state
-        //var state = container.getState();
-        //long filledQuantity = state.getChildOrders().stream().map(ChildOrder::getFilledQuantity).reduce(Long::sum).get();
+        var state = container.getState();
+        long filledQuantity = state.getChildOrders().stream().map(ChildOrder::getFilledQuantity).reduce(Long::sum).get();
 
         //and: check that our algo state was updated to reflect our fills when the market data
-        //assertEquals(225, filledQuantity);
+        assertEquals(100, filledQuantity);
```

Comment:
does this test now pass?

Reply:
Yeah, I did not write that test. it came with the codebase.